dotcom-rendering/src/model/enhanceCards.ts

```diff
@@ -163,6 +164,7 @@ const enhanceTags = (tags: FETagType[]): TagType[] => {
 
 export const enhanceCards = (
 	collections: FEFrontCard[],
+	editionId?: EditionId,
```

Comment:
What scenario would we not have sectionId in? Could this be a non optional param?

Reply:
Yeah, I'll make it non-optional. Makes sense! I was ambivalent myself but thought I'll leave this up to the reviewer!

Reply:
Ah actually I remembered why I didn't make it non-optional! 

The DOM structure is:
`FrontLayout` > `FrontSection` > `ShowMore` ([we call `enhanceCards` in `ShowMore.importable.tsx`](https://github.com/guardian/dotcom-rendering/blob/7ebe8b895744871cd8333c5c7c630ebfb7525eee/dotcom-rendering/src/web/components/ShowMore.importable.tsx#L116-L117))

[`editionId` in `FrontSection` is optional](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/web/components/FrontSection.tsx#L52) and all the props of this component. I am note sure about why this is the case. If we want to make `editionId` non-optional in the `enhanceCards` method, we'll have to do the same all the way up, otherwise Typescript is complaining that I'm trying to pass an optional value to a non-optional parameter.

Making it non-optional requires changes to all the components where `FrontSection` is used. I am happy to do this refactoring but I think it will be better in a separate PR!

Reply:
Ahhh, yea I remember this too now... A lot of the props are optional in FrontSection so that you don't need them to be set in Storybook!